packages/react-components/react-datepicker-compat/library/src/components/DatePicker/useDatePicker.tsx

```diff
@@ -364,7 +364,6 @@ export const useDatePicker_unstable = (props: DatePickerProps, ref: React.Ref<HT
 
   const inputRoot = slot.always(props.root, {
     defaultProps: {
-      'aria-owns': open ? popupSurfaceId : undefined,
```

Comment:
just confirming that this was intended :)

Reply:
yup! This is the fix for the OH-GOD-FIRE bug in the original issue 😅

This prop made sense when the dialog was non-modal (it needed to be inline in the a11y tree), but now that it's modal, it moves the dialog to the hidden a11y tree, making it inaccessible :P